src/js/utils/VtxDeviceStatus/SmartAudioDeviceStatus.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+class VtxDeviceStatusSmartAudio extends VtxDeviceStatus {
+    constructor(dataView)
+    {
+        super(dataView);
+
+        dataView.readU8(); // custom device status size
+
+        this._version = dataView.readU8();
+        this._mode = dataView.readU8();
+        this._orfreq = dataView.readU16();
+        this._willBootIntoPitMode = Boolean(dataView.readU8());
+    }
+
+    get smartAudioVersion()
+    {
+        const sa = this._version * 100 + this._mode;
```

Comment:
Kind of a weird approach to first merge two settings into one variable, and then do a switch over it. ;-)